addon-test-support/-private/ember-exam-qunit-test-loader.js

```diff
@@ -107,6 +108,17 @@ export default class EmberExamQUnitTestLoader extends TestLoader {
     super.unsee(moduleName);
   }
 
+  /**
+   * setupModuleMetadataHandler() register QUnit callback to enable generating module metadata detail file.
```

Comment:
nitpick: `setupModuleMetadataHandler() register QUnit callback to enable generating module metadata file.`